docs/en/index.md

```diff
@@ -2,246 +2,66 @@
-[refer-to-this-documentation]: https://developer.ubuntu.com/core/get-started/developer-setup
-[get-in-contact]: https://www.ubuntu.com/core/contact-us
-[guide]: https://docs.ubuntu.com/core/en/build-store/create
-[step-by-step-guide]: https://snapcraft.io/docs/build-snaps/publish
-[please-see-the-full-snap-documentation]: https://docs.snapcraft.io/snaps/
-
+Ubuntu Core is Ubuntu for IoT and embedded environments, optimised for security and
+reliable updates. It's easy to deploy, tamper-resistant, and hardened against
+corruption.
+
+Its read-only root filesystem is built from the same packages used to build the
+wider set of Ubuntu distributions, and only differs in the way packages are
+delivered, and crucially, updated. This is all handled by snaps, a secure,
+confined, dependency-free, cross-platform Linux packaging system.
+
+Snap packages ensure there is always a clean separation between the base system
+and whatever applications need to be installed, as well as isolation between
+each and every application, their data, and even application version data.
+
+Updates are transactional, which means they’re either 100% successful or
+they’re not installed. If they're not installed, they leave no trace of any
+failure other than log details. This means the system remains fully operational
+and in a continuously well-defined state during both application and system
+updates. The system can also recover or revert to previous states if necessary, 
+even if a system fails to boot. Unlike alternative or more traditional
+package managers, a failed update never leaves the system in an unpredictable
+state.
+
+Ubuntu Core is engineered to meet the precise requirements of IoT devices. It
+runs on a large range of hardware, including Raspberry Pi, Intel NUC, Qualcomm
+Snapdragon 410c and even a KVM. Features include:
+
+- **simple, consistent installation and deployment**: Ubuntu Core is installed
+  via an immutable image, which can be either installed or built specifically
+  for your platform and application
+- **a read-only filesystem**: apps run in isolation from each other and access
+  to system resources is only granted with explicit permissions
+- **transactional updates**: signed, autonomous and atomic, updates can withstand
+  unpredictable hardware and network conditions, even to the operating system
+- **snap-based, security first**: snaps are secure, easy to build, and painless
+  to distribute. Public/private key validation ensures what's running is
+exactly what's intended to run
+
+Designed from the ground-up to solve the complex problems associated with
+deploying, running and updating critical software, **Ubuntu Core** makes updates
+automatics, reliable, secure and transparent whilst still offering a broad and
+carefully selected level of control and customisation over an update’s
+deployment.
+
+#### Ubuntu Core and Ubuntu
+
+Ubuntu is widely known as the Ubuntu Desktop and Ubuntu Server Linux
```

Comment:
Seems a bit clumsy. Maybe, "The Ubuntu family of distributions is widely known from" or includes or something.